HelpSource/Classes/ServerTreeView.schelp

```diff
@@ -7,7 +7,7 @@ description::
 link::Classes/ServerTreeView:: shows the node tree on a server.
 
 note::
-The link::Classes/ServerTreeView:: will disappear when the pointed server is not running.
+Unlikely to link::Classes/Server#-plotTree::, the link::Classes/ServerTreeView:: will not disappear when the pointed server is not running.
```

Comment:
`Unlikely to link::Classes/Server#-plotTree::,` -> `Unlike link::Classes/Server#-plotTree::,`

Reply:
Thanks, done in https://github.com/supercollider/supercollider/pull/6834/commits/bf4cc47da33fa088117afc433197068e265af4db